ci-operator/step-registry/idp/external-oidc/keycloak/aks/server/idp-external-oidc-keycloak-aks-server-commands.sh

```diff
@@ -0,0 +1,492 @@
+oc apply -f https://github.com/jetstack/cert-manager/releases/download/v1.13.0/cert-manager.yaml
+oc wait --for=condition=ready pod -l app.kubernetes.io/instance=cert-manager -n cert-manager --timeout=300s
+# Create Let's Encrypt ClusterIssuer
+cat <<EOF | oc apply -f -
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-staging
+spec:
+  acme:
+    server: https://acme-staging-v02.api.letsencrypt.org/directory
+    email: openshift-qe@redhat.com
+    privateKeySecretRef:
+      name: letsencrypt-staging
+    solvers:
+    - http01:
+        ingress:
+          class: nginx
+EOF
+oc wait --for=condition=ready clusterissuer letsencrypt-staging --timeout=120s
+
+# Set Keycloak repo
+/tmp/linux-amd64/helm repo add bitnami https://charts.bitnami.com/bitnami
+/tmp/linux-amd64/helm repo update
+
+# Create Keycloak settings file
+cat > /tmp/keycloak-values.yaml << EOF
+nameOverride: ""
+fullnameOverride: ""
+namespace: keycloak
+
+auth:
+  adminUser: ${KEYCLOAK_ADMIN_TEST_USER}
+  adminPassword: "${KEYCLOAK_ADMIN_TEST_PASSWORD}"
+image:
+  # Use the bitnamilegacy repo, if we meet docker pulling limit issue, 
+  # will try to update use quay.io image
+  registry: docker.io
+  repository: bitnamilegacy/keycloak
+  tag: 26.3.3-debian-12-r0
+  pullPolicy: IfNotPresent
+
+securityContext:
+  enabled: true
+  fsGroup: 1001
+  runAsUser: 1001
+  runAsNonRoot: true
+
+envFrom:
+  - secretRef:
+      # This secret stores the admin-password of keycloak
+      name: "keycloak"
+
+extraVolumes:
+  - name: setup-scripts
+    configMap:
+      name: "keycloak-setup-scripts"
+      defaultMode: 0755
+
+extraVolumeMounts:
+  - name: setup-scripts
+    mountPath: /opt/bitnami/keycloak/setup-scripts/
+    readOnly: true
+
+lifecycleHooks:
+  postStart:
+    exec:
+      command:
+        - /bin/bash
+        - -c
+        - |
+          #!/bin/bash
+          set -euo pipefail
+
+          echo "Waiting for Keycloak server to be ready..."
+          timeout 10m bash -c 'while true; do
+            echo "Checking Keycloak health at http://localhost:${KC_HTTP_PORT:-8080}/health/ready" >> /tmp/postStart.log
+            if curl -v http://localhost:${KC_HTTP_PORT:-8080}/health/ready >> /tmp/postStart.log 2>&1; then
+              echo "Keycloak server is ready. Starting initialization..." >> /tmp/postStart.log
+              break
+            fi
+            echo "Keycloak not ready yet, retrying in 5 seconds..." >> /tmp/postStart.log
+            sleep 5
+          done' || {
+            echo "ERROR: Timeout waiting for Keycloak server after 10 minutes. Check /tmp/postStart.log for details" >> /tmp/postStart.log
+            exit 1
+          }
+
+          echo "Executing setup-script.sh..." >> /tmp/postStart.log
+          /opt/bitnami/keycloak/setup-scripts/setup-script.sh &>> /tmp/postStart.log || {
+            echo "ERROR: setup-script.sh failed. Check /tmp/postStart.log for details"
+            exit 1
+          }
```

Comment:
Like https://github.com/openshift/release/blob/master/ci-operator/step-registry/idp/external-oidc/keycloak/server/idp-external-oidc-keycloak-server-commands.sh#L71 , put all shell scripts into setup-script.sh. We can huddle for why.